codegen/builtin_fs.go

```diff
@@ -69,6 +69,7 @@ func (i Image) Call(ctx context.Context, cln *client.Client, ret Register, opts
 		case llb.ImageOption:
 			imageOpts = append(imageOpts, o)
 		case *specs.Platform:
+			platform = *o
```

Comment:
Nit: move the definition of `resolveOpt` down so we don't need the line below?

Reply:
fixed